### DISCUSSION TITLE
Failed Assert BR-BT-00514-0304 (Warning)

### DISCUSSION THREAD

**Discussion by AlexAS81** [OP-TED/eForms-SDK#900] [Q&A] [answered]
What does this warning really imply? Would manual validations leave the notice as NOT PUBLISHED or would it be published correctly?

It is a notice without European funding and with a company from outside the European Union.

`<svrl:failed-assert id="BR-BT-00514-0304" location="/cn:ContractNotice/ext:UBLExtensions/ext:UBLExtension/ext:ExtensionContent/efext:EformsExtension/efac:Organizations/efac:Organization[3]/efac:Company" test="((../../../../../../../cac:ProcurementProjectLot[cbc:ID/@schemeName='Lot']/cac:TenderingTerms/cbc:FundingProgramCode[@listName='eu-funded']/normalize-space(text()) = 'eu-funds') or (cac:PostalAddress/cac:Country/cbc:IdentificationCode[../../../cac:PartyIdentification/cbc:ID/normalize-space(text()) = ../../../../../../../../../../cac:ContractingParty/cac:Party/cac:PartyIdentification/cbc:ID/normalize-space(text())]/normalize-space(text()) = ('ALB','ATF','AUT','BEL','BGR','BLM','CHE','CPT','CYP','CZE','DEU','DNK','ESP','EST','FIN','FRA','GRC','HRV','HUN','IRL','ISL','ITA','LIE','LTU','LUX','LVA','MDA','MKD','MLT','MNE','NCL','NLD','NOR','POL','PRT','PYF','ROU','SPM','SRB','SVK','SVN','SWE','TUR','UKR','WLF'))) or not((cac:PartyIdentification/cbc:ID/normalize-space(text()) = ../../../../../../../cac:ContractingParty/cac:Party/cac:PartyIdentification/cbc:ID/normalize-space(text())) and not(../../../efac:NoticeSubType/cbc:SubTypeCode/normalize-space(text()) = ('1','2','3','4','5','6','E1','E2')))" role="WARN" flag="LAWFULNESS">
<svrl:text>'The procurement is at least partially financed by European Union funds' (BT-60-Lot) or the buyer must be in a country for which the publication is expressly authorised</svrl:text>
<svrl:diagnostic-reference diagnostic="ND-Company_BT-514-Organization-Company" see="field:BT-514-Organization-Company">
<svrl:text>cac:PostalAddress/cac:Country/cbc:IdentificationCode</svrl:text>
</svrl:diagnostic-reference>
</svrl:failed-assert>`

Thanks in advance,
Alex

**Comment by YvesJo** [comment, marked as answer]
Hi,
If this error occurs, then manual control will take place within a given period. When not rejected, it should get published.

**Reply by AlexAS81** [reply]
I do not understand well. Would it be rejected or not?

**Reply by YvesJo** [reply]
When the manual control confirms this is not something that should get published, then its publication will get rejected.
